crates/ty_python_semantic/src/types/signatures.rs

```diff
@@ -617,6 +603,15 @@ impl<'db> Signature<'db> {
         inferable: InferableTypeVars<'_, 'db>,
         visitor: &IsEquivalentVisitor<'db>,
     ) -> ConstraintSet<'db> {
+        // The typevars in self and other should also be considered inferable when checking whether
+        // two signatures are equivalent.
+        let self_inferable =
+            (self.generic_context).map(|generic_context| generic_context.inferable_typevars(db));
+        let other_inferable =
+            (other.generic_context).map(|generic_context| generic_context.inferable_typevars(db));
```

Comment:
(obviously not blocking, just curious) -- are the parentheses around `self.generic_context` to try to get rustfmt to format this the way you'd like it to be formatted?